server/data/incentivesApi.ts

```diff
@@ -1,6 +1,37 @@
+// eslint-disable-next-line max-classes-per-file
 import config from '../config'
 import RestClient from './restClient'
 
+export class ErrorResponse {
+  status: number
+
+  errorCode?: ErrorCode
+
+  userMessage?: string
+
+  developerMessage?: string
+
+  moreInfo?: string
+
+  static isErrorResponse(obj: object): obj is ErrorResponse {
+    // TODO: would be nice to make userMessage & developerMessage non-nullable in the api
+    return obj && 'status' in obj && typeof obj.status === 'number'
+  }
+}
+
+export enum ErrorCode {
```

Comment:
Do you reckon is worth adding a note referring to the enum in the API where these are defined?

Reply:
done